lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -65,25 +79,20 @@ def code
       end
 
       # REQUIRED - This is the http status code.
-      # unless you've specified that you want the status code to be something other
-      # then 400 explicitly it will default to 400. IT WILL NOT DEFAULT to whatever
-      # was provided by the backend service, because the backend service response
-      # might not always be relevant
       def status
-        i18n_data[:status].presence || 400
+        i18n_data[:status] || 400
```

Comment:
I might need more info on this error's role. For a downstream 500 lately, a Faraday::Error (which becomes a Common::Client::Errors::ClientError), I figure out what the VA system error really means and then throw a specific error class, either a 403, 422, 503, or 504. The error then bubbles up directly to the controller or it's caught somewhere up the chain, where it can be rescued as is without additionally having to check the code. TLDR; I don't actually use this error so feel free to ignore :) maybe because it's too generic?